PocketCoffea/workflows/base.py

```diff
@@ -116,7 +109,7 @@ def load_metadata(self):
         self._year = self.events.metadata["year"]
         self._triggers = triggers[self.cfg.finalstate][self._year]
         self._btag = btag[self._year]
-        self._isMC = self.events.metadata["isMC"]
+        self._isMC = self.events.metadata["isMC"] == "True"
```

Comment:
Is this a string at the end in the dataset definition?

Reply:
Probably better to be sure in build_dataset to have a boolean..

Reply:
Yes, at the moment the attribute `isMC` is a string. This needs to be fixed when building the json dataset.